verifier/src/report_body.rs

```diff
@@ -203,6 +247,21 @@ mod test {
         assert_eq!(verifier.verify(&report_body).is_some().unwrap_u8(), 1);
     }
 
+    #[test]
+    fn report_body_fails_due_to_config_security_version() {
+        let report_body = ReportBody::from(&REPORT_BODY_SRC);
+        let mut expected_config_svn = report_body.config_svn();
+        *expected_config_svn.as_mut() += 1;
```

Comment:
Why not
```suggestion
        let expected_config_svn = report_body.config_svn() + 1;
```

Reply:
I updated to
```rust
        let expected_config_svn = *report_body.config_svn().as_mut() + 1;
```
The `ConfigSvn` doesn't support adding. I don't think it's much of a use case outside of testing so not adding a tracking item.